test/cases/libvirt.sh

```diff
@@ -2,19 +2,21 @@
 set -euo pipefail
 
 # Get OS data.
-source /etc/os-release
-DISTRO_CODE="${DISTRO_CODE:-${ID}_${VERSION_ID//./}}"
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
 
 # Provision the software under test.
 /usr/libexec/osbuild-composer-test/provision.sh
 
 # Test the images
 /usr/libexec/osbuild-composer-test/libvirt_test.sh qcow2
 
-/usr/libexec/osbuild-composer-test/libvirt_test.sh openstack
+#TODO: remove this condition once there is rhel9 support for openstack and vhd image types
+if [[ "$DISTRO_CODE" != rhel-90 ]]; then
+  /usr/libexec/osbuild-composer-test/libvirt_test.sh openstack
+fi
```

Comment:
@ondrejbudai I'm not sure if this is like it is supposed to be or it's a mistake from resolving conflicts. Can you take a look?

Reply:
No longer needed.